File: 2020/test/day_20_test.exs
defmodule Day20Test do
  use ExUnit.Case

  import Day20
  import TestHelper

  describe "Day20" do
    test "part1" do
      assert part1(input("day_20_example_1")) == 5
      assert part1(input("day_20")) == 2436
    end

    # test "part2" do
    #   assert part2(input("day_18_example_1")) == 231
    #   assert part2(input("day_18_example_2")) == 51
    #   assert part2(input("day_18_example_3")) == 693_891
    #   assert part2(input("day_18")) == 224_973_686_321_527
    # end
  end
end

File: 2021/test/day_20_test.rb
# frozen_string_literal: true

require "test_helper"
require "day_20"

class Day20Test < Minitest::Test
  def test_part1
    day = Day20.new

    assert_equal 35, day.part1(input("day_20_example_1"))
    assert_equal 5306, day.part1(input("day_20"))
  end

  def test_part2
    day = Day20.new

    assert_equal 3351, day.part2(input("day_20_example_1"))
    assert_equal 17_497, day.part2(input("day_20"))
  end
end
